src/shared/components/network.tsx

```diff
@@ -0,0 +1,61 @@
+import { useNetwork } from "@starknet-react/core";
+import { cn } from "@cartridge/ui";
+import { Chain } from "@starknet-react/chains";
+import { useCallback } from "react";
+import { getChecksumAddress } from "starknet";
+import { toast } from "sonner";
+import React from "react";
+import { useScreen } from "@/shared/hooks/useScreen";
+
+const ChainColors: Record<Chain["network"], string> = {
+  mainnet: "bg-[#FF4264]",
+  sepolia: "bg-[#6DE27C]",
+  slot: "bg-[#FBCB4A]",
+  localhost: "bg-[#9C9C9C]",
```

Comment:
on second thought, let's convert this as a fallback for 'other' networks.

```suggestion
  other: "bg-[#9C9C9C]",
```